pnpm-lock.yaml

```diff
@@ -42181,6 +42283,39 @@ packages:
       yargs-parser: 21.1.1
     dev: true
 
+  /ts-jest/29.1.1_gvmt7aj7nai5bnvsxmiksfugce:
```

Comment:
Why isn't the prior `/ts-jest/29.1.1_...` going away?
I see the difference is removal of `@bable/core` dependency, but don't relate that back to the vue changes.?

Reply:
I reset the lockfile to the main branch version and reran pnpm i, but this dupe is still there. Don't know why.